2_process.R

```diff
@@ -178,6 +178,16 @@ p2_targets_list <- list(
                      cat_attr_list = p2_cat_attr_list,
                      vaa_cols = c("SLOPE"),
                      sites_w_segs = p2_sites_w_nhd_segs)
+ ),
+
+
+ tar_target(
+   p2_met_data_at_obs_sites,
+   {
+     reticulate::source_python("2_process/src/subset_nc_to_comid.py")
+     as_tibble(subset_nc_to_comids(p1_drb_nhd_gridmet, p2_med_observed_sites$COMID))
```

Comment:
`p2_med_observed_sites` is a vector of character strings, so I'm confused where the `$COMID` reference is coming from here.

Reply:
I think this should actually be `p2_med_observed_reaches$COMID`. Although it still looks like I'm missing one COMID: 
```
* start target p2_met_data_at_obs_sites
x error target p2_met_data_at_obs_sites
* end pipeline: 5.33 seconds
Error : AssertionError: 

Detailed traceback:
  File "<string>", line 34, in subset_nc_to_comids

[4782491]
x Problem with the pipeline.
i Show errors: tar_meta(fields = error, complete_only = TRUE)
i Learn more: https://books.ropensci.org/targets/debugging.html
Error:
! problem with the pipeline.
```

Reply:
Confirming that this runs for me if I change to `p2_med_observed_reaches$COMID` _and_ if I do some local edits such that `p1_drb_nhd_gridmet` is pointing to a local copy of the full-DRB gridmet data. So `4782491` must just be missing from Jeff's 6/14/22 subset file. 

```
> tar_load(p2_met_data_at_obs_sites)
> head(p2_met_data_at_obs_sites)
# A tibble: 6 x 10
    COMID  tmmx  tmmn    pr  srad    vs  rmax  rmin     sph time               
    <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl>   <dbl> <dttm>             
1 4492036  61.9  47.8 0.481  35.8  9.57  86.1  58.5 0.00628 1978-12-31 19:00:00
2 4495680  59.3  48.8 0.632  35.6  7.70  82.8  66.0 0.00686 1978-12-31 19:00:00
3 4495656  58.4  48.0 0.634  35.7  7.63  84.2  66.9 0.00678 1978-12-31 19:00:00
4 4784831  59.9  48.9 0.610  35.5  7.77  82.2  65.5 0.00690 1978-12-31 19:00:00
5 4782619  58.1  48.3 0.625  35.5  7.61  84.8  67.5 0.00679 1978-12-31 19:00:00
6 4782585  57.3  46.7 0.610  36.0  7.57  88.8  69.0 0.00673 1978-12-31 19:00:00
>
```